packages/commerce/backend/src/app.ts

```diff
@@ -62,6 +63,12 @@ export class App {
     app.use(helmet())
     app.use(express.json())
     app.use(express.urlencoded({ extended: true, limit: '25mb' }))
+    app.use(
+      '/assets',
```

Comment:
assets is confusing in context of testnet, can this be changed to `images` instead?